### ISSUE TITLE
Title in tabs doesn't work

### ISSUE THREAD

**Issue by PixelT** [AndiDittrich/WordPress.Enlighter#34]
I paste to wordpress the codegroup code from the example in plugin description:
```
[codegroup]
    [js tab="Javascript Message"]
    window.addEvent('domready', function(){
        // display string on console
        console.info('Hello Enlighter');

        // show element
        $('#myelement').show();
    }); 
    [/js]

    [html]
    <div id="myelement">
    INITIALIZATION START
    </div>      
    [/html]

    [css tab="Styling"]
    #myelement{
        color: #cc2222;
        padding: 15px;
        font-size: 20px;
        text-align: center;     
    }       
    [/css]  
[/codegroup]
```

Instead of defined tab title, I have language name in title...

**Comment by AndiDittrich** [comment]
Dear PixelT,

thanks for your report - i can confirm the issue within both shortcode processors. 
I've accidentally changed the related html attribute name in v2.0

**Comment by AndiDittrich** [comment]
Fixed in v3.1
